scss/list.scss

```diff
@@ -0,0 +1,26 @@
+.books-container {
+  flex-wrap: wrap;
+  max-width: 1000px;
+  width: 100%;
+}
+
+.book {
+  background: $white;
+  margin: 0 36px 36px 0;
+  width: 198px;
+}
+
+.book-info {
+  margin: 28px;
```

Comment:
Careful with this. Remember, we only use margin-right and margin-bottom for responsive reasons. Additionally we only use them to separate siblings. If you need to separate your content from your container use padding instead. This is making a strange behavior 

![image](https://user-images.githubusercontent.com/36488701/68787486-65829c00-0620-11ea-924b-822599efcefa.png)

It's working but in an unpredictable way. Use padding instead. 

It's also 'cause your index to behave differently: 

![image](https://user-images.githubusercontent.com/36488701/68787589-9531a400-0620-11ea-845e-c5deeef90887.png)